helm/acapy-cloud/conf/local/driver-did-cheqd.yaml

```diff
@@ -0,0 +1,31 @@
+fullnameOverride: driver-did-cheqd
+
+replicaCount: 1
+
+podAnnotations:
+  sidecar.istio.io/proxyCPU: 10m
+
+image:
+  registry: ghcr.io
+  name: cheqd/did-registrar
+  pullPolicy: Always
+  tag: production-latest
```

Comment:
Waiting for the Cheqd team to pull multi-arch CI changes from https://github.com/cheqd/did-resolver/pull/363 and do a release